openai/cli.py

```diff
@@ -508,6 +511,29 @@ def help(args):
     )
     sub.set_defaults(func=FineTune.prepare_data)
 
+    sub = subparsers.add_parser("search.prepare_data")
+    sub.add_argument(
+        "-f",
+        "--file",
+        required=True,
+        help="JSONL, JSON, CSV, TSV, TXT or XLSX file containing prompt-completion examples to be analyzed."
+        "This should be the local file path.",
+    )
+    sub.add_argument(
+        "-p",
+        "--purpose",
+        help="Why are you uploading this file? (see https://beta.openai.com/docs/api-reference/ for purposes)",
```

Comment:
It seems unintuitive to me that the `Search` purpose (which is what this is ostensibly under) is further split into different purposes. Maybe each of them should be a different tool instead? (Or do they all fall under the search umbrella, and how? We can chat outside this PR if i'm missing some context)